src/renderers/shaders/ShaderLib.js

```diff
@@ -225,14 +225,27 @@ const ShaderLib = {
 
 	},
 
+	backgroundCube: {
```

Comment:
Instead of using the term `backgroundCube`, what do you think about `skybox` instead?

Reply:
OK, we can change it. How do you feel about `blurredCube` shader? That is the most descriptive name I can think of...

Reply:
True, but I have the feeling `skybox` is what most 3D users understand of what the shader essentially represents. It's a common term in 3D engines.

However, the name is not something that blocks the PR for me. If we can't quickly agree on a name, we can merge the refactoring first and then discuss alternatives. `backgroundCube` isn't wrong after all.

Reply:
We already have a skybox shader called 'cube' shader.

- 'cube': a skybox shader, used by post-processing, and other apps

- 'background': used by the renderer for `scene.background` (flat only)

- 'backgroundCube': used by the renderer for blurring `scene.background` (cube only)

The nomenclature is not a deal breaker for me, but let's revisit in a later PR. 😇